package.json

```diff
@@ -25,9 +25,11 @@
 	],
 	"scripts": {
 		"build": "tsup src/index.ts --clean --format cjs,esm --outDir lib --dts",
+		"docs": "typedoc",
 		"format": "prettier \"**/*\" --ignore-unknown",
 		"format:write": "pnpm format --write",
 		"lint": "eslint . --max-warnings 0 --report-unused-disable-directives",
+		"lint:docs": "typedoc --validation --treatWarningsAsErrors",
```

Comment:
[Request] These two new commands should be mentioned in `.github/DEVELOPMENT.md`. `docs` is pretty essential and there's an existing list of `lint:*` commands.